central/clusters/zip/render_test.go

```diff
@@ -80,6 +82,7 @@ func doTestRenderOpenshiftEnv(t *testing.T, clusterType storage.ClusterType) {
 }
 
 func TestRenderWithNoCollection(t *testing.T) {
+	t.Setenv(defaults.ImageFlavorEnvName, defaults.ImageFlavorNameRHACSRelease)
```

Comment:
Let's be consistent and use `ImageFlavorNameDevelopmentBuild` in the same file, unless `ImageFlavorNameRHACSRelease` is necessary for this test to pass.
```suggestion
	t.Setenv(defaults.ImageFlavorEnvName, defaults.ImageFlavorNameDevelopmentBuild)
```